misc/grimoirelab2sh

```diff
@@ -112,32 +108,33 @@ def to_json(uidentities, organizations, source):
         uuid = uidentity.uuid
 
         uid = uidentity.to_dict()
-        #uid['identities'].sort(key=lambda x: x['email'])
+        uid['identities'].sort(key=lambda x: x['email'])
```

Comment:
Remove this call because `email` field is not required. When I run the script I get the next error when some email entry is not present:
```
Traceback (most recent call last):
  File "grimoirelab2sh", line 157, in <module>
    main()
  File "grimoirelab2sh", line 50, in main
    j = to_json(parser.identities, parser.organizations, args.source)
  File "grimoirelab2sh", line 111, in to_json
    uid['identities'].sort(key=lambda x: x['email'])
TypeError: unorderable types: str() < NoneType()
```